src/components/Modal/helper.js

```diff
@@ -16,6 +19,7 @@ export const getContentHeight = ({
 
   return `${container.clientHeight -
     (isFullscreen ? 0 : SPACING * 2) -
+    (isWebview ? NATIVE_APP_NAV_BAR_HEIGHT : 0) -
```

Comment:
I would suggest we allow for a custom offsetTop to be passed, rather than hardcoding a variant for webview usage.

Reply:
I've updated the implementation and added the `overlayProps` prop to pass custom styling to the overlay. But in any case, we need to adjust calculation logic to keep the content scrolling work.

Reply:
Looks great! Thanks @dtereshenko